style.css

```diff
@@ -144,14 +144,29 @@ body {
 }
 
 .note {
+  animation: fadein 0.7s;
```

Comment:
```suggestion
  animation: fadein 0.3s;
```
Decrease this to 0.3 seconds as this speed up the total time.

Reply:
I've completed the task. I hope its all good now.